migrations/202107061359_create_webhooks.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS webhooks (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(60) NOT NULL,
```

Comment:
name is 60 here and 100 on validation. I think 60 is enough, so just align the number there please

Reply:
Damn, that's a lot of inconsistency... I must be tired at the time I did this 😆